.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
This template should be removed. Feature requests don't make sense in a docs repo